src/measure_theory/constructions/borel_space.lean

```diff
@@ -428,6 +428,129 @@ lemma is_preconnected.measurable_set
   (h : is_preconnected s) : measurable_set s :=
 h.ord_connected.measurable_set
 
+lemma exists_dense_seq_lt {α : Type*} [nonempty α] [topological_space α] [separable_space α]
+  [linear_order α] [order_topology α] (hnlb : ∀ x : α, ∃ y, y < x) (x : α) :
```

Comment:
This lemma and the next one should go to `topology/order/basic`. Assumption `hnlb` is called `no_bot_order α`.

Reply:
I assume you mean `topology/algebra/ordered/basic`?